color/src/palette/css.rs

```diff
@@ -0,0 +1,297 @@
+// Copyright 2024 the Color Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! The CSS named colors.
+//!
+//! The classic set of CSS named colors, also known as the [X11] colors.
+//!
+//! These are all of type [`AlphaColor<Srgb>`](AlphaColor).
```

Comment:
I thought I read somewhere that using these is generally recommended against, because they don't work together coherently, and so more carefully designed palletes should be used. But I can't find that documented anywhere "authoratitive".